pkg/tfpfbridge/tests/internal/cmd/pulumi-resource-testbridge/schema.json

```diff
@@ -81,6 +81,21 @@
                 "handlers",
                 "port"
             ]
+        },
+        "testbridge:index/TestresSingleNestedAttr:TestresSingleNestedAttr": {
+            "properties": {
+                "description": {
+                    "type": "string"
+                },
+                "quantity": {
+                    "type": "number"
+                }
+            },
+            "type": "object",
+            "required": [
```

Comment:
The optional/required distinction is lost due to an underlying issue in TFPF mapping types to TF types. Perhaps we could work around this and recover the info that is still present in the TPFF-only attr types.

Reply:
This would be problematic for updating random and other providers no?

Reply:
It didn't show in the random schema diff, but it will bite at some point, I'll file an issue to fix it up.